test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -170,6 +174,62 @@ constexpr int calc_res_size(int n, int) { return n; }
 auto data_gen2_default = [](auto i) { return i % 5 ? i : 0;};
 auto data_gen_zero = [](auto) { return 0;};
 
+// Extension: host execution policy type trait
+template <class _T>
+struct __is_host_execution_policy : std::false_type
```

Comment:
Nitpick: there is no need to uglify in the tests

Reply:
Fixed